lms/djangoapps/instructor_task/tests/test_integration.py

```diff
@@ -293,7 +294,7 @@ def test_rescoring_bad_unicode_input(self):
 
         # return an input error as if it were a numerical response, with an embedded unicode character:
         expected_message = u"Could not interpret '2/3\u03a9' as a number"
-        with patch('capa.capa_problem.LoncapaProblem.rescore_existing_answers') as mock_rescore:
+        with patch('capa.capa_problem.LoncapaProblem.get_grade_from_current_answers') as mock_rescore:
             mock_rescore.side_effect = StudentInputError(expected_message)
```

Comment:
@jcdyer this integration test expects the a CAPA-specific `StudentInputError` and validates that the output of the overall task hasn't changed.